src/main/java/org/example/services/ImportedItemTaxService.java

```diff
@@ -0,0 +1,26 @@
+package org.example.services;
+
+import org.example.constants.TaxRateConstants;
+
+public class ImportedItemTaxService implements TaxCalculationInterface {
```

Comment:
check if we can use abstract design pattern? also write in pr description why is this better

Reply:
Yes abstract factory method can also be used here , i have refactored the code to use it , added its benefit in pr